helms/odahu-flow-packagers/templates/packers/cli.yaml

```diff
@@ -0,0 +1,102 @@
+{{- if .Values.packagers.cli.enabled -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: "{{ .Release.Name }}-cli-packager-resources"
+  labels:
+    {{- include "odahuflow.helm-labels" (dict "component" "fluentd" "root" .) | nindent 4 }}
+data:
+  resources.yaml: |
+    kind: PackagingIntegration
```

Comment:
You can combine the two yaml definition of Packaging Integration in a one config map. Thereof we don't need two repeat the job definition two times.